README.md

```diff
@@ -1 +1,106 @@
-# webpacktodolist
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<img src="./microverse.png" alt="logo" width="140"  height="auto" />
+<br/>
+
+<h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+  - [🚀 Live Demo](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [💻 Getting Started](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Setup](setup)
+  - [Prerequisites](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Install](install)
+  - [Usage](html,javascript,css)
+- [👥 Authors](https://github.com/PierreBarba)
+- [🔭 Future Features](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [🤝 Contributing](https://github.com/cilfonegabriel)
+- [⭐️ Show your support](support)
+- [🙏 Acknowledgements](https://github.com/elafildecolor,https://github.com/luigirazum)
+- [📝 License](MIT)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capton Project] <a name="Todo List"></a>
+
+
+**[Personal Portfolio Pierre]** is a...
+
```

Comment:
- [ ] Great job using Microverse README template 💯 but I think you forgot to add the project name, and description, so Kindly add them 👍

Reply:
thanks for the comment